src/Core/Data/SqlRedirectRepository.cs

```diff
@@ -0,0 +1,228 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Common;
+using BVNetwork.NotFound.Core.CustomRedirects;
+using EPiServer.Data;
+using EPiServer.Logging;
+
+namespace BVNetwork.NotFound.Core.Data
+{
+    public class SqlRedirectRepository : IRepository<CustomRedirect>, IRedirectLoader
+    {
+        private static readonly ILogger Logger = LogManager.GetLogger();
```

Comment:
cant this be passed in trough constructor?

Reply:
does it need to be static?

Reply:
Episerver's convention is this as I know. Haven't seen injected logger.